shared/app/nav.desktop.js

```diff
@@ -22,16 +21,11 @@ type Props = {
   routePath: I.List<string>,
 }
 
-const hotkeyTabMap: {[string]: Tabs.Tab} = {
-  '1': Tabs.peopleTab,
-  '2': Tabs.chatTab,
-  '3': Tabs.fsTab,
-  '4': Tabs.teamsTab,
-  '5': Tabs.devicesTab,
-  '6': Tabs.gitTab,
-  '7': Tabs.settingsTab,
-  ...(flags.walletsEnabled ? {'8': Tabs.walletsTab} : {}),
-}
+
+const hotkeyTabMap = Tabs.desktopTabOrder.reduce((tabMap, tab, index) => {
```

Comment:
Anyone want to try to find a shorter way to do this? :)  `const hotkeyTabMap = {...Tabs.desktopTabOrder}` is close -- but it's 0-indexed instead of 1-indexed.

Reply:
1. you can keep it zero based and just change it in the _onHotkey to add the offset
1. you could just do something like `{...['', ...Tabs.desktopTabOrder]}` but thats not very readable